client/components/keyed-suggestions/index.jsx

```diff
@@ -251,10 +253,26 @@ class KeyedSuggestions extends React.Component {
 				// Concatenate mathing and non matchin - this is full set of filters just reordered.
 				filtered[ key ] = [ ...sortedMatching, ...notMatching ];
 			} else {
+				// Matcher is designed to be used with args (term.name, cleanFilterTerm)
+				// Order is important!
+				// Arg 1 can be multiple words. "flexible header" or "accepts header images of any size"
+				// Arg 2 will only be one word; even if the user types multiple words we search on each one individually.
+				const matcher = ( term1, term2_single, threshold = SEARCH_THRESHOLD ) => {
```

Comment:
We can remove the `threshold` arg and use `SEARCH_THRESHOLD` directly below, no?